src/pthread-main.js

```diff
@@ -0,0 +1,96 @@
+// Pthread Web Worker startup routine:
```

Comment:
it seems confusing for `pthread-main.js` to be the main code run first by the pthreads. The name could also be understood as the main of all the pthreads, I think.